RELEASE_NOTES.md

```diff
@@ -2,85 +2,58 @@
 
 ## Summary
 
-This version ships an experimental version of the **Power Manager**, adds preliminary support for n:m relations between inverters and batteries and includes user documentation.
+The `microgrid` package now exposes grid connections uniformly and introduces formula operators for `consumption` and `production`, replacing the `logical_meter.*_{production,consumption}()` formulas. The `actor` package restarts crashed actors with a delay, and the `ConnectionManager` exposes the `microgrid_id` and `location` details.
+
+There are also a few bug fixes, documentation improvements and other minor breaking changes.
 
 ## Upgrading
 
-- `microgrid.battery_pool()` method now accepts a priority value.
+- `actor` package
 
-- `microgrid.grid()`
+  * Actors are now restarted after a small delay when they crash to avoid a busy loop and spamming the logs if the actor keeps failing to start.
 
-  * Similar to `microgrid.battery_pool()`, the Grid is now similarily accessed.
+- `microgrid` package
 
-- `BatteryPool`'s control methods
+  * `grid`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
 
-  * Original methods `{set_power/charge/discharge}` are now replaced by `propose_{power/charge/discharge}`
-  * The `propose_*` methods send power proposals to the `PowerManagingActor`, where it can be overridden by proposals from other actors.
-  * They no longer have the `adjust_power` flag, because the `PowerManagingActor` will always adjust power to fit within the available bounds.
-  * They no longer have a `include_broken_batteries` parameter.  The feature has been removed.
+  * `BatteryPool`: The `include_broken_batteries` argument was removed from the control methods. This option is no longer supported.
```

Comment:
I think this belongs to `- BatteryPool's control methods` which has been removed and not to `microgrid package`

Reply:
You mean because the `BateryPool` object is defined in `timeseries`?

Reply:
Nevermind, I've just understood the changes done

Reply:
Ouch, please don't approve if you see any issues and auto-merge is enabled :laughing:

Reply:
I looked at the commit and see this is not exposed via `microgrid.battery_pool()`, so OK, will put it outside the `microgrid` package.

Reply:
OK, this was very hard to track what was really changed and which parts were affected. @frequenz-floss/python-sdk-team when writing release notes we should think more from the user's perspective at the time to upgrade to write good notes, useful notes.

Reply:
Updated to hopefully really reflect the user-visible changes and in a useful way.

Reply:
not sure what the confusion is,  the flag was part of the battery pools control methods,  and they've gone away: https://frequenz-floss.github.io/frequenz-sdk-python/v0.25/reference/frequenz/sdk/timeseries/battery_pool/#frequenz.sdk.timeseries.battery_pool.BatteryPool.charge

this was not about power distributing actor internals.

Reply:
It affects the access via `microgrid.battery_pool()` (which IMHO is the main thing the user cares about) and also some other public (although probably not used by users really) interfaces.